src/RouteConfig.js

```diff
@@ -87,8 +99,7 @@ export default (
             (argumentName: string): string => argumentName.replace(':', ''),
           );
           const values = argumentNames
-            .map((argument: string): string => request.params[argument])
-            .filter((value: ?Object): boolean => value !== undefined);
```

Comment:
why were we need this? It breaks optional params in routes. Without this filter, optional params work, and seems other things still work.